Kaiju druid - Jay

JayPea00101010 opened the pull request:
3rd times the charm right?

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -0,0 +1,90 @@
+		</setters>
+	</element>
+	
+	<element name="Circle of The Strange Beasts" type="Archetype" source="Reddit" id="ID_RDDT_ARCHETYPE_CIRCLE_OF_THE_STRANGE_BEASTS">
```
I'd set source to "Reddit: Unearthed Arcana" to differentiate between different subreddits.

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -0,0 +1,90 @@
+			<p>When you join this circle at 2nd level, you become far more sensitive to monstrosities very existence due to your privileged link with them. You can take 1 minute to focus, allowing you to sense the presence and general location of monstrosities and circle of Strange beasts druids within a 1000ft radius around you for 10 minutes. You can use this feature a number of times equal to your Wisd ...
+		</description>
+		<sheet>
+			<description>When you join this circle at 2nd level, you become far more sensitive to monstrosities very existence due to your privileged link with them. You can take 1 minute to focus, allowing you to sense the presence and general location of monstrosities and circle of Strange beasts druids within a 1000ft radius around you for 10 minutes. You can use this feature a number of times equal to ...
```
"circle of Strange beasts druids" - Capitalise? "Circle of Strange Beasts Druids"?

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -0,0 +1,90 @@
+			<p>Starting at 6th level, you always have the enlarge/reduce spell prepared, and it doesn't count against your number of prepared spells per day. Also, when you use your Wild shape feature, you can use a free action to immediatly spend a 2nd level or higher spell slot to cast it on yourself.</p>
+		</description>
+		<sheet>
+			<description>You always have the enlarge/reduce spell prepared, and it doesn't count against your number of prepared spells per day. Also, when you use your Wild shape feature, you can use a free action to immediatly spend a 2nd level or higher spell slot to cast it on yourself.</description>
```
"immediatly" should be "immediately"

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -0,0 +1,90 @@
+			<p>Starting at 2nd level, you can choose, during a long rest, a monstrosity that you have seen before with a challenge rating of 1 or lower. You can use your Wild shape feature to assume this creature's shape. Once you've used your Wild shape feature this way, you must finish a long rest before using it in this manner again. You can choose another monstrosity to turn into once in a number of d ...
+		</description>
+		<sheet>
+			<description>You can choose, during a long rest, a monstrosity that you have seen before with a challenge rating of 1 or lower. You can use your Wild shape feature to assume this creature's shape. Once you've used your Wild shape feature this way, you must finish a long rest before using it in this manner again. You can choose another monstrosity to turn into once in a number of days equal to  ...
```
I believe it's possible to hide the "at 6th level" bit until you're 6th level...

Can anyone else confirm?

JayPea00101010 replied:
Do you mean line 64?

JayPea00101010 replied:
oh right ignore the previous comment (If that actually sent)

ColdAnkles replied:
Do you want to leave this open to investigate this? or close it and leave it for later?

JayPea00101010 replied:
I've figured out how to do it, about to edit now

JayPea00101010 commented:
Adjusted capitalization of "Circle of The Strange Beasts Druids" in lines 51 and 48

Changed all instances of: source="Reddit"
To: source="Reddit: Unearthed Arcana"

corrected typo of "immediatly" in lines 64 and 67

JayPea00101010 commented:
And I've added a new line (60) that'll change what is shown on the sheet at level 6

JayPea00101010 commented:
also, I'm not sure how to add the change you requested

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -56,9 +56,10 @@
 		<sheet>
-			<description>You can choose, during a long rest, a monstrosity that you have seen before with a challenge rating of 1 or lower. You can use your Wild shape feature to assume this creature's shape. Once you've used your Wild shape feature this way, you must finish a long rest before using it in this manner again. You can choose another monstrosity to turn into once in a number of days equal to  ...
+			<description>You can choose, during a long rest, a monstrosity that you have seen before with a challenge rating of 1 or lower. You can use your Wild shape feature to assume this creature's shape. Once you've used your Wild shape feature this way, you must finish a long rest before using it in this manner again. You can choose another monstrosity to turn into once in a number of days equal to  ...
+			<description level="6">You can choose, during a long rest, a monstrosity that you have seen before with a challenge rating of 1 or lower. You can use your Wild shape feature to assume this creature's shape. Once you've used your Wild shape feature this way, you must finish a long rest before using it in this manner again. You can choose another monstrosity to turn into once in a number of days ...
```
"your druid level divided by 2, rounded down"

You could add this as a stat - <stat name="monster week:cr" value="level:druid:half" />

something like that

JayPea00101010 replied:
pretty sure there was a conversation on the discord that you can't do half values, could be wrong, I'll double check

JayPea00101010 replied:
nevermind, looks like you can

swdriessen commented on reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml:
```diff
@@ -0,0 +1,91 @@
+			<p>When you join this circle at 2nd level, you become far more sensitive to monstrosities very existence due to your privileged link with them. You can take 1 minute to focus, allowing you to sense the presence and general location of monstrosities and Circle of Strange Beasts Druids druids within a 1000ft radius around you for 10 minutes. You can use this feature a number of times equal to yo ...
+		</description>
+		<sheet>
+			<description>When you join this circle at 2nd level, you become far more sensitive to monstrosities very existence due to your privileged link with them. You can take 1 minute to focus, allowing you to sense the presence and general location of monstrosities and Circle of Strange Beasts Druids within a 1000ft radius around you for 10 minutes. You can use this feature a number of times equal to ...
```
You can leave out information you don't need for sheet descriptions to make them shorter.
_When you join this circle at 2nd level..._

And you can create statistics to replace sheet descriptions such as _'You can use this feature a number of times equal to your Wisdom modifier (a minimum of once).'_ Check out some other files on how to do this, almost all files have them.

JayPea00101010 replied:
I changed that line but I'm not sure how to make it minimum 2

JayPea00101010 replied:
*minimum 1

JayPea00101010 replied:
ok changed it so that's fine now

JayPea00101010 commented:
Changed the file name and all references to it from "druid-kaiju" to "druid-strangebeasts"

Cut down sheet descriptions

Commented out the source

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes/druid-strangebeasts.xml:
```diff
@@ -5,11 +5,11 @@
 	</info>
 
-	<element name="Reddit: Strange Beasts" type="Source" source="Core" id="ID_RDDT_SOURCE_STRANGE_BEASTS">
+	<!-- <element name="Reddit: Strange Beasts" type="Source" source="Core" id="ID_RDDT_SOURCE_STRANGE_BEASTS"> -->
```
You need to comment out the rest of the Source element. Just commenting the opening breaks stuff.

JayPea00101010 replied:
oh right yeah, that's really bad on my part

JayPea00101010 commented:
Actually correctly commented out the source

ColdAnkles commented:
I don't think you need to set version to 0.0.4 before it's committed to the repo

JayPea00101010 commented:
ah right, so just leave it as 0.01?

JayPea00101010 commented:
OK everything should be fine now (I hope)

ColdAnkles commented on reddit/reddit-unearthed-arcana/archetypes.index:
```diff
@@ -2,13 +2,14 @@
 		<file name="monk-fourelementsremastered.xml" url="https://raw.githubusercontent.com/community-elements/elements-reddit/master/reddit/reddit-unearthed-arcana/archetypes/monk-fourelementsremastered.xml" />	
-		
+	<!-- Circle of the Strange Beasts -->
+		<file name="druid-strangebeasts.xml" url="https://raw.githubusercontent.com/community-elements/elements-reddit/master/reddit/reddit-unearthed-arcana/archetypes/druid-kaiju.xml" />			
```
the URL still points to the old version of the name.

Once fixed, I'll merge.

JayPea00101010 replied:
oh oof, sorry

JayPea00101010 commented:
I've fixed the URL in the index file now
